projects/services-ogc/assets/exampleContext.ts

```diff
@@ -409,3 +409,473 @@ export const exampleContext: IOwsContext = {
+              }]
+          },
+          "type": "Feature",
+          "id": "OpenLayers_Layer_WMS_133883",
+          "geometry": {
+              "type": "Polygon",
+              "coordinates": [[
+                  [
+                      -178.21502685547,
+                      18.924781799316
+                  ],
+                  [
+                      -178.21502685547,
+                      71.406646728516
+                  ],
+                  [
+                      -66.969848632813,
+                      71.406646728516
+                  ],
+                  [
+                      -66.969848632813,
+                      18.924781799316
+                  ],
+                  [
+                      -178.21502685547,
+                      18.924781799316
+                  ]
+              ]]
+          }
+      },
+      {
+          "properties": {
+              "title": "BlueMarbleCov",
+              "updated": "2016-09-30T16:18:30Z",
+              "content": "BlueMarbleCov",
+              "categories": [
+                  {
+                      "term": true,
+                      "scheme": "http://www.opengis.net/owc/active"
+                  },
+                  {
+                      "term": 1,
+                      "scheme": "http://www.envitia.com/horizon/layer/opacity"
+                  }
+              ],
+              "offerings": [{
+                  "code": "http://www.opengis.net/spec/owc-geojson/1.0/req/wcs",
+                  "operations": [
+                      {
+                          "code": "GetCapabilities",
+                          "method": "GET",
+                          "type": "text/xml",
+                          "href": "http://ows.rasdaman.org/rasdaman/ows?REQUEST=GetCapabilities&SERVICE=WCS&VERSION=2.0.0"
+                      },
+                      {
+                          "code": "GetCoverage",
+                          "method": "GET",
+                          "href": "http://ows.rasdaman.org/rasdaman/ows?SCALEFACTOR=10.174297058223718&format=image/png&CoverageId=BlueMarbleCov&request=GetCoverage&version=2.0.0&service=WCS"
+                      }
+                  ]
+              }]
+          },
+          "type": "Feature",
+          "id": "OpenLayers_Layer_Image_134365",
+          "geometry": {
+              "type": "Polygon",
+              "coordinates": [[
+                  [
+                      -180,
+                      -90
+                  ],
+                  [
+                      -180,
+                      90
+                  ],
+                  [
+                      180,
+                      90
+                  ],
+                  [
+                      180,
+                      -90
+                  ],
+                  [
+                      -180,
+                      -90
+                  ]
+              ]]
+          }
+      }
+  ]
+};
+
+
+export const wuekisContext: IOwsContext = {
```

Comment:
Is it ok for **wuekis** to add this context to our library?

Reply:
Good point! Has been adjusted - now using publicly available server.